src/main.rs

```diff
@@ -412,12 +394,26 @@ fn gen_block(args: GenBlock) -> Result<()> {
 }
 #[derive(Default, serde::Serialize, serde::Deserialize)]
 struct Config {
-    transforms: Vec<chiptool::transform::Transform>,
+    includes: Option<Vec<String>>,
```

Comment:
for both `includes` and `transforms`, `None` and `Some([])` mean the same thing, so the Option is a bit redundant.

To make the fields optional, you can add `#[serde(default)]` instead of the option, so it defaults to an empty Vec if the user doesn't specify a value.